specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-05-01/mfe.json

```diff
@@ -0,0 +1,10065 @@
+      ],
+      "properties": {
+        "compute": {
+          "description": "Compute target for batch inference operation.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "errorThreshold": {
+          "format": "int32",
+          "description": "Error threshold, if the error count for the entire input goes above this value,\r\nthe batch inference will be aborted. Range is [-1, int.MaxValue].\r\nFor FileDataset, this value is the count of file failures.\r\nFor TabularDataset, this value is the count of record failures.\r\nIf set to -1 (the lower bound), all failures during batch inference will be ignored.",
+          "default": -1,
+          "type": "integer"
+        },
+        "loggingLevel": {
+          "description": "Logging level for batch inference operation.",
+          "default": "Info",
+          "$ref": "#/definitions/BatchLoggingLevel"
+        },
+        "maxConcurrencyPerInstance": {
+          "format": "int32",
+          "description": "Indicates maximum number of parallelism per instance.",
+          "default": 1,
+          "type": "integer"
+        },
+        "miniBatchSize": {
+          "format": "int64",
+          "description": "Size of the mini-batch passed to each batch invocation.\r\nFor FileDataset, this is the number of files per mini-batch.\r\nFor TabularDataset, this is the size of the records in bytes, per mini-batch.",
+          "default": 10,
+          "type": "integer"
+        },
+        "model": {
+          "description": "Reference to the model asset for the endpoint deployment.",
+          "$ref": "#/definitions/AssetReferenceBase",
+          "x-nullable": true
+        },
+        "outputAction": {
+          "description": "Indicates how the output will be organized.",
+          "default": "AppendRow",
+          "$ref": "#/definitions/BatchOutputAction"
+        },
+        "outputFileName": {
+          "description": "Customized output file name for append_row output action.",
+          "default": "predictions.csv",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Provisioning state for the endpoint deployment.",
+          "$ref": "#/definitions/DeploymentProvisioningState",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "resources": {
+          "description": "Indicates compute configuration for the job.\r\nIf not provided, will default to the defaults defined in ResourceConfiguration.",
+          "$ref": "#/definitions/ResourceConfiguration",
+          "x-nullable": true
+        },
+        "retrySettings": {
+          "description": "Retry Settings for the batch inference operation.\r\nIf not provided, will default to the defaults defined in BatchRetrySettings.",
+          "$ref": "#/definitions/BatchRetrySettings",
+          "x-nullable": true
+        }
+      },
+      "x-ms-client-name": "BatchDeploymentDetails",
```

Comment:
```suggestion
      "x-ms-client-name": "BatchDeploymentProperties",
```

Reply:
all changes requested in this review should be addressed - resolving all conversations